Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -606,13 +608,16 @@ protected override void RenderContents (HtmlTextWriter writer)
     }
 
     /// <summary> Renders the <paremref name="nodes"/> onto the <paremref name="writer"/>. </summary>
-    private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection nodes, bool isTopLevel, HashAlgorithm nodeIDAlgorithm, int nestingDepth)
+    private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection webTreeNodes, bool isTopLevel, HashAlgorithm nodeIDAlgorithm, int nestingDepth)
```

Comment:
We should clean up the formatting in this method.

 - remove leading empty line
 - personally I would split up the `nodes` ternary into multiple lines `cond \r\n ? ... \r\n : ...`
 - single statement ifs should still have a line break